app/assets/stylesheets/print.scss

```diff
@@ -1,5 +1,4 @@
 @import 'govuk_publishing_components/govuk_frontend_support';
-@import 'govuk_publishing_components/components/print/feedback';
```

Comment:
Is this intentional? We do have it in for media screen.

Reply:
I seem to be having a bit of a morning. No, it's not intentional... will fix.